routing/pathfind.go

```diff
@@ -506,6 +507,10 @@ func findPath(g *graphParams, r *RestrictParams, source, target Vertex,
 		ignoredNodes = make(map[Vertex]struct{})
 	}
 
+	// We use this map to track which nodes have already been visited so
```

Comment:
I would have said something like "explored all possible incoming channels", since "visited" is ambiguous .